org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/typesystem/xpect/numbers.wlk.xt

```diff
@@ -107,7 +107,7 @@ object numberTest {
 		return value.isInteger()
 	}
 
-	// XPECT methodType at m26 --> (Number) => Number
+	// XPECT methodType at m26 --> ((Number|keyboard)) => (Key|Number)
```

Comment:
What if we rename this to plusSign in order to avoid colission?
Or we can rename the plus in Number to unaryPlus... it is not used a lot.

Reply:
@fdodino ^

Reply:
I changed the keyboard methods to plusKey() and minusKey(). That are not used too.